static/js/taxpayer_creation_form_logic.js

```diff
@@ -82,45 +82,33 @@ $("#id_address_form-country").change(function(){
 function arg_is_ri(){
```

Comment:
NIT: I know that this is not from this PR but it will be good a more expressive name